core-bundle/contao/templates/twig/content_element/accordion.html.twig

```diff
@@ -25,7 +25,7 @@
 {% block script %}
     {% add "handorgel_js" to body %}
         <script src="{{ asset('js/handorgel.min.js', 'contao-components/handorgel') }}"></script>
```

Comment:
And don‘t we have to add a nonce here, too?

Reply:
And what about here?

https://github.com/contao/contao/blob/c25426d7b2bf84c4c86860d18533d4c5b0b5cceb/core-bundle/templates/Error/layout.html.twig#L11

Reply:
Here yes. These templates did not exist yet when I did my CSP PR (or at least when I started it), that's why it was still missing I guess.